lib/adiwg/mdtranslator/readers/mdJson/modules/module_plan.rb

```diff
@@ -0,0 +1,86 @@
+require_relative 'module_citation'
+
+module ADIWG
+    module Mdtranslator
+        module Readers
+            module MdJson
+  
+                module Plan
+                    def self.unpack(hPlan, responseObj, inContext = nil)
+                        
+                        @MessagePath = ADIWG::Mdtranslator::Readers::MdJson::MdJson
+
+                        # return nil object if input is empty
+                        if hPlan.empty?
+                            @MessagePath.issueWarning(40, responseObj, inContext)
+                            return nil
+                        end
+
+                        intPlan = intMetadataClass.newPlan
+
+                        statusArray = [
+                            "accepted",
+                            "cancelled",
+                            "completed",
+                            "deprecated",
+                            "final",
+                            "funded",
+                            "historicalArchive",
+                            "notAccepted",
+                            "obsolete",
+                            "onGoing",
+                            "pending",
+                            "planned",
+                            "proposed",
+                            "required",
+                            "retired",
+                            "superseded",
+                            "suspended",
+                            "tentative",
+                            "underDevelopment",
+                            "valid",
+                            "withdrawn"
+                        ]
```

Comment:
This is a code list right? There's a module that should be used to get the code list and validate the code is part of it.